Cython/Utility/NumpyImportArray.c

```diff
@@ -0,0 +1,15 @@
+///////////////////////// NumpyImportArray.init ////////////////////
+
+// comment below is deliberately kept in the generated C file to
+// help users debug where this came from:
+/*
+ * Cython has automatically inserted a call to _import_array since
+ * you didn't include one when your cimported numpy. To disable this
+ * add the line
+ *   <void>_import_array
+ */
+#ifdef NPY_NDARRAYOBJECT_H /* numpy headers have been included */
+#if !NO_IMPORT_ARRAY /* can be explicitly set to tell Numpy its initialization is done manually */
```

Comment:
```suggestion
#if ! CYTHON_NO_IMPORT_ARRAY /* can be explicitly set to tell Numpy its initialization is done manually */
```

Reply:
I don't think I'd do this. I was trying to detect people using https://docs.scipy.org/doc/numpy-1.17.0/reference/c-api.array.html#c.NO_IMPORT_ARRAY directly - not provide an extra way to override it.

Reply:
Ok, I didn't know that macro. Makes sense to me, but I found the comment to be misleading. It read like this was something that Cython users could use as an override, whereas it's actually a NumPy feature. Maybe add the documentation link to the comment in the file?